src/js/users/components/Permission.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import styled from "styled-components";
+import { BoxGroupSection, Icon } from "../../base";
+
+const StyledPermissionIcon = styled(Icon)`
+    margin-right: 15px;
+`;
+
+const StyledPermissionItem = styled(BoxGroupSection)`
+    align-items: center;
+    display: flex;
+`;
+
+type PermissionItemProps = {
+    /* Name of the permission */
+    permission: string;
+    /* Indicates if the permission is granted */
+    value: boolean;
+};
+
+/**
+ * Permission item with an icon indicating its status.
+ * @prop permission - name of the permission.
+ * @prop value - indicates if the permission is granted.
```

Comment:
I don't think `@props` is part of the tsdoc standard. Also information here is redundant with the information the the type